<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyMedDialog</class>
 <widget class="QDialog" name="BuyMedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuyMedDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Billing</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Medicine:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="medicine_lineedit"/>
      </item>
      <item>
       <widget class="QPushButton" name="search_button">
        <property name="text">
         <string>search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Available qt:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="avai_quant_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>MRP (for each):   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mrp_label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="quantity_sb"/>
      </item>
      <item>
       <widget class="QPushButton" name="addbutton">
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_button">
        <property name="text">
         <string>DELETE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="allitems_table">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Item ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MRP</string>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Total Items:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="t_items_label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>TOTAL VALUE:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="show_tvalue">
        <property name="text">
         <string>0  Rs</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Customer ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="custid_lineedit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Customer Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="custname_linedit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="generatebill_button">
        <property name="text">
         <string>generate bill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
